primitives/npos-elections/src/lib.rs

```diff
@@ -349,13 +349,40 @@ pub struct Support<AccountId> {
 	pub voters: Vec<(AccountId, ExtendedBalance)>,
 }
 
+#[cfg(feature = "mocks")]
+impl<AccountId: Ord + Clone> Support<AccountId> {
+	/// `true` when the support is identical except for the ordering of the voters.
+	pub fn eq_unordered(&self, other: &Self) -> bool {
+		self.total == other.total && {
+			let my_voters: BTreeMap<_, _> = self.voters.iter().cloned().collect();
+			let other_voters: BTreeMap<_, _> = other.voters.iter().cloned().collect();
+			my_voters == other_voters
+		}
+	}
+}
+
 /// A target-major representation of the the election outcome.
 ///
 /// Essentially a flat variant of [`SupportMap`].
 ///
 /// The main advantage of this is that it is encodable.
 pub type Supports<A> = Vec<(A, Support<A>)>;
 
+#[cfg(feature = "mocks")]
+pub fn supports_eq_unordered<AccountId: Ord + Clone>(
```

Comment:
@kianenigma these look unused - any idea if its ok to rmeove?

Reply:
if you merge master they should be removed, else remove them + the feature `mock`